Gemfile

```diff
@@ -59,7 +58,7 @@ group :test do
 end
 
 group :production do
-  gem 'mysql2', '~> 0.3.20'
+  gem 'mysql2', '~> 0.4.4'
```

Comment:
Does this actually work now? We had to pin before because activerecord 4.x wasn't happy with it.